### PR TITLE
display a message to inform the user that their registry is empty

### PR DESCRIPTION
**Pull request by expravit**
Resolves Issue #1022

### CODE REVIEW

File: src/homepage/PrototypesPage.js [hunk 1]
```diff
@@ -10,6 +10,20 @@ import type {Assets} from "../webutil/assets";
...
+        <li key={stringify(x)}>
+          <Link to={`/prototypes/${x.repoId.owner}/${x.repoId.name}/`}>
+            {`${x.repoId.owner}/${x.repoId.name}`}
+          </Link>
+        </li>
+      ))}
+    </ul>
+  );
+  const displayWarning = (
+      <p>You haven't loaded any data into the registry yet!</p>
```
Comment (decentralion) [inline src/homepage/PrototypesPage.js]: So... the use of the single tick freaks the GitHub syntax higlighter out. Channeling @wchargin here, I think we should fix it by using the unicode apostrophe symbol `'` (https://www.fileformat.info/info/unicode/char/27/index.htm)

Comment (wchargin) [reply src/homepage/PrototypesPage.js]: The Unicode U+0027 APOSTROPHE _is_ the single tick that you find on your keyboard. You want U+2019 RIGHT SINGLE QUOTATION MARK `’` (Vim: `<C-k>'9`).

But yes, use that.

Comment (wchargin) [reply src/homepage/PrototypesPage.js]: You can always visit <http://rsquo.net> if you forget it. :-)

**Comment by decentralion** [comment]
Also, @expravit: please add a modification to the top of CHANGELOG.md mentioning the bugfix.

**Comment by decentralion** [comment]
@expravit I'm closing this as inactive for now, feel free to ping and re-open it if you want to continue on it.

**Pull request closed without merging**